ldms/src/test/Makefile.am

```diff
@@ -41,10 +41,14 @@ test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
 # override pkglib sanity checks
 mypkglibdir = $(pkglibdir)
-mypkglib_SCRIPTS = ldms-run-static-tests.sh
+mypkglib_SCRIPTS = ldms-run-static-tests.test
 mypkglib_DATA = ldms-static-test-list.sh ldms-static-test-bypass
 
+# invoke tap-driver
+TEST_LOG_DRIVER = env AM_TAP_AWK='$(AWK)' $(SHELL) \
+                  $(top_srcdir)/config/tap-driver.sh
+
 installcheck-local: 
-	ldms-run-static-tests.sh
+	ldms-run-static-tests.test
```

Comment:
make check works.

A minor improvement on 'make check' with tap added would be to give the count of tests skipped (and possibly show them as SKIP in the output); but this would be additional work, not a reason to hold up this PR.

make installcheck on a fresh checkout fails with:
```
make[3]: Leaving directory `/home/baallan/ovis/repos/o4sara/LDMS_objdir/ldms/src/sampler'
Making installcheck in test
make[3]: Entering directory `/home/baallan/ovis/repos/o4sara/LDMS_objdir/ldms/src/test'
ldms-run-static-tests.test
make[3]: ldms-run-static-tests.test: Command not found
make[3]: *** [installcheck-local] Error 127
make[3]: Leaving directory `/home/baallan/ovis/repos/o4sara/LDMS_objdir/ldms/src/test'
make[2]: *** [installcheck-recursive] Error 1
make[2]: Leaving directory `/home/baallan/ovis/repos/o4sara/LDMS_objdir/ldms/src'
make[1]: *** [installcheck-recursive] Error 1
make[1]: Leaving directory `/home/baallan/ovis/repos/o4sara/LDMS_objdir/ldms'
make: *** [installcheck-recursive] Error 1
```

Is install-check-local a holdover from previous development, or is the file it's trying to call missing?

Reply:
I believe the file naming was wrong. The installcheck hook was looking explicitly for a file named "ldms-run-static-tests.test" in that directory, however the file was actually "ldms-run-static-tests.test**.in**. I made this change and make installcheck worked for me.